image/tarexport/save.go

```diff
@@ -393,6 +423,11 @@ func (s *saveSession) saveImage(id image.ID) (map[layer.DiffID]distribution.Desc
 	var layers []layer.DiffID
 	var foreignSrcs map[layer.DiffID]distribution.Descriptor
 	for i, diffID := range img.RootFS.DiffIDs {
+		select {
```

Comment:
Should we check this down at L581 as well? (sorry GH won't let me add a comment there)

Reply:
Not sure - it's calling `system.Chtimes` on two paths. Doesn't look like it has a potential to take a noticeable amount of time.